public/app/features/annotations/annotations_srv.ts

```diff
@@ -195,4 +210,64 @@ export class AnnotationsSrv {
   }
 }
 
+export function executeAnnotationQuery(
```

Comment:
this is an internal function now -- but when we revisit the annotations query path (and expose it to react panels!), we may want to expose `AnnotationQueryResponse` in grafana/data